packages/molenc/molenc.5.0.0/opam

```diff
@@ -17,7 +17,7 @@ install: [
 depends: [
   "bst" {>= "2.0.0"}
   "dune" {>= "1.11"}
-  "batteries"
+  "batteries" {>= "2.9.0"}
   "dolog" {>= "4.0.0" & < "5.0.0"}
   "minicli"
```

Comment:
```suggestion
  "minicli" {>= "5.0.0"}
```
Looking at the code, I think this became wrapped with dunification in v 5.
Should solve
```
#=== ERROR while compiling molenc.5.0.0 =======================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.04.2 | pinned(https://github.com/UnixJunkie/molenc/archive/v5.0.0.tar.gz)
# path                 ~/.opam/4.04/.opam-switch/build/molenc.5.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p molenc -j 31
# exit-code            1
# env-file             ~/.opam/log/molenc-7-2f3223.env
# output-file          ~/.opam/log/molenc-7-2f3223.out
### output ###
# File "/home/opam/.opam/4.04/lib/bst/bst.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
#       ocamlc src/.encoder.eobjs/byte/decoder.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.04/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.encoder.eobjs/byte -I /home/opam/.opam/4.04/lib/batteries -I /home/opam/.opam/4.04/lib/bst -I /home/opam/.opam/4.04/lib/bytes -I /home/opam/.opam/4.04/lib/dolog -I /home/opam/.opam/4.04/lib/minicli -I /home/opam/.opam/4.04/lib/num -I /home/opam/.opam/4.04/lib/ocaml/threads -I /home/opam/.opam/4.04/lib/parmap -I src/.molenc.objs/byte -I src/.molenc.objs/native -no-alias-deps -o src/.encoder.eobjs/byte/decoder.cmo -c -impl src/decoder.ml)
# File "src/decoder.ml", line 15, characters 13-24:
# Error: Unbound module Minicli
#       ocamlc src/.encoder.eobjs/byte/encoder.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.04/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.encoder.eobjs/byte -I /home/opam/.opam/4.04/lib/batteries -I /home/opam/.opam/4.04/lib/bst -I /home/opam/.opam/4.04/lib/bytes -I /home/opam/.opam/4.04/lib/dolog -I /home/opam/.opam/4.04/lib/minicli -I /home/opam/.opam/4.04/lib/num -I /home/opam/.opam/4.04/lib/ocaml/threads -I /home/opam/.opam/4.04/lib/parmap -I src/.molenc.objs/byte -I src/.molenc.objs/native -no-alias-deps -o src/.encoder.eobjs/byte/encoder.cmo -c -impl src/encoder.ml)
# File "src/encoder.ml", line 18, characters 13-24:
# Error: Unbound module Minicli
#       ocamlc src/.encoder.eobjs/byte/filter.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.04/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.encoder.eobjs/byte -I /home/opam/.opam/4.04/lib/batteries -I /home/opam/.opam/4.04/lib/bst -I /home/opam/.opam/4.04/lib/bytes -I /home/opam/.opam/4.04/lib/dolog -I /home/opam/.opam/4.04/lib/minicli -I /home/opam/.opam/4.04/lib/num -I /home/opam/.opam/4.04/lib/ocaml/threads -I /home/opam/.opam/4.04/lib/parmap -I src/.molenc.objs/byte -I src/.molenc.objs/native -no-alias-deps -o src/.encoder.eobjs/byte/filter.cmo -c -impl src/filter.ml)
# File "src/filter.ml", line 18, characters 13-24:
# Error: Unbound module Minicli
#       ocamlc src/.encoder.eobjs/byte/uniq.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.04/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.encoder.eobjs/byte -I /home/opam/.opam/4.04/lib/batteries -I /home/opam/.opam/4.04/lib/bst -I /home/opam/.opam/4.04/lib/bytes -I /home/opam/.opam/4.04/lib/dolog -I /home/opam/.opam/4.04/lib/minicli -I /home/opam/.opam/4.04/lib/num -I /home/opam/.opam/4.04/lib/ocaml/threads -I /home/opam/.opam/4.04/lib/parmap -I src/.molenc.objs/byte -I src/.molenc.objs/native -no-alias-deps -o src/.encoder.eobjs/byte/uniq.cmo -c -impl src/uniq.ml)
# File "src/uniq.ml", line 6, characters 13-24:
# Error: Unbound module Minicli
#       ocamlc src/.encoder.eobjs/byte/pubchem_decoder.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.04/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.encoder.eobjs/byte -I /home/opam/.opam/4.04/lib/batteries -I /home/opam/.opam/4.04/lib/bst -I /home/opam/.opam/4.04/lib/bytes -I /home/opam/.opam/4.04/lib/dolog -I /home/opam/.opam/4.04/lib/minicli -I /home/opam/.opam/4.04/lib/num -I /home/opam/.opam/4.04/lib/ocaml/threads -I /home/opam/.opam/4.04/lib/parmap -I src/.molenc.objs/byte -I src/.molenc.objs/native -no-alias-deps -o src/.encoder.eobjs/byte/pubchem_decoder.cmo -c -impl src/pubchem_decoder.ml)
# File "src/pubchem_decoder.ml", line 18, characters 13-24:
# Error: Unbound module Minicli
#       ocamlc src/.encoder.eobjs/byte/butina.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.04/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.encoder.eobjs/byte -I /home/opam/.opam/4.04/lib/batteries -I /home/opam/.opam/4.04/lib/bst -I /home/opam/.opam/4.04/lib/bytes -I /home/opam/.opam/4.04/lib/dolog -I /home/opam/.opam/4.04/lib/minicli -I /home/opam/.opam/4.04/lib/num -I /home/opam/.opam/4.04/lib/ocaml/threads -I /home/opam/.opam/4.04/lib/parmap -I src/.molenc.objs/byte -I src/.molenc.objs/native -no-alias-deps -o src/.encoder.eobjs/byte/butina.cmo -c -impl src/butina.ml)
# File "src/butina.ml", line 19, characters 13-24:
# Error: Unbound module Minicli
```